course-details.md

```diff
@@ -1,9 +1,43 @@
-This course is about continuous delivery, or CD, with GitHub Actions. We create two workflows to deploy our app to an AWS S3 bucket automatically. CD is an important part of modern software development, and it has a big meaning. CD is the practice of delivering software faster and with higher quality. To do this, it takes many different practices, behaviors, and technologies. 
+This course is about continuous delivery, or CD, with GitHub Actions. CD is a crucial part of modern software development, and it can have a monumental impact on development projects. 
 
-In this course, we focus on workflows to deploy pull requests automatically to a staging environment, and to deploy all merged commits to production. In this course, you will:
+Continuous Delivery is the practice of delivering software faster and with higher quality. Doing so successfully requires many different practices, behaviors, and technologies. 
 
-- Create two workflow files
+In this course, we focus on using workflows to deploy pull requests automatically to a staging environment, and to deploy all merged commits to production. We will create two workflows to deploy our app to an AWS S3 bucket automatically.
+
+
+## What you'll learn
+
+We'll answer common questions like:
+- What is Continuous Delivery?
+- What is a workflow file and how are workflows used with GitHub Actions?
+- What are deployment triggers and how are they used with deployments?
+- How can secrets store tokens?
+
+
+And when you're done you'll be able to:
+- Use GitHub Actions and create workflow files
 - Configure AWS S3 for deployment
 - Use secrets to store tokens
 - Deploy to staging and production
-- Practice using GitHub Actions
\ No newline at end of file
+
+
+## What you'll build
+In this course, we'll be walking through a CD deployment using GitHub Actions, following steps such as adding conditional triggers:
+
+![cd](https://user-images.githubusercontent.com/57373296/75816734-bea74480-5d63-11ea-8f3c-d4b93680d1bf.gif)
+
+## Prerequisites
+Before you start, you should be familiar with GitHub and Continuous Integration. We recommend taking the following two Learning Labs prior to this course:
+
+- [Introduction to GitHub](https://lab.github.com/githubtraining/introduction-to-github)
+- [Continuous Integration with GitHub Actions](https://lab.github.com/githubtraining/set-up-continuous-integration-with-github-actions)
```

Comment:
How about the packages course? I think we can leave the Intro to GitHub course out if we link to other courses that require it.

Reply:
I think it would be best to reference "Hello GitHub Actions", the CI course, and the upcoming (not built yet) workflow course